src/state/slices/stakingDataSlice/stakingDataSlice.ts

```diff
@@ -15,18 +21,38 @@ export type Staking = {
   undelegations: chainAdapters.cosmos.Undelegation[]
   rewards: chainAdapters.cosmos.ValidatorReward[]
 }
+
+export type Validators = {
+  validators: chainAdapters.cosmos.Validator[]
+}
+
 export type StakingDataByAccountSpecifier = {
-  [k: CAIP10]: Staking
+  [k: PubKey]: Staking
```

Comment:
[gomesalexandre](https://github.com/gomesalexandre) [7 hours ago](https://github.com/shapeshift/web/pull/1356/files#r840746380)
That's a pubkeyish, which in the case of cosmos is a CAIP10 but this is not always going to be the case, I'd just type this as string for now or keep the CAIP10

Reply:
agree

Reply:
Still relevant, reopening